.github/workflows/test-action-tuf.yaml

```diff
@@ -21,9 +21,9 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.22.x
         - v1.23.x
         - v1.24.x
+        - v1.25.x
```

Comment:
This doesn't work with the v0.4.6 version below because that didn't support 1.25. I'd reckon drop the 1.25 test here for now, or change the v0.4.6 to test with something later (which will require changes in the community). Maybe easier to just drop this test for now, I can add it back later and clean up which versions we test with?